components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -110,11 +111,19 @@ public void addEmailTemplateType(String emailTemplateDisplayName, String tenantD
     public void addNotificationTemplateType(String displayName, String notificationChannel, String tenantDomain)
             throws NotificationTemplateManagerException {
 
+        addNotificationTemplateType(displayName, notificationChannel, tenantDomain, null);
+    }
+
+    @Override
+    public void addNotificationTemplateType(String displayName, String notificationChannel,
```

Comment:
Check if email template interface can be a direct extension point from notification